frontend/src/app/shared/types/enums/ButtonState.ts

```diff
@@ -0,0 +1,5 @@
+export enum ButtonState {
+  Visible_Enabled,
```

Comment:
typescript conventions would be `VisibleEnabled` change if you agree, but this conventions is also fine for me :+1:

Reply:
angepasst